src/signals/IncidentMap/components/utils/get-filtered-incidents.ts

```diff
@@ -51,7 +51,7 @@ export const getFilteredIncidents = (
 
 interface Icon {
   slug: string
-  icon: string
+  icon?: string
```

Comment:
ow die zat er nog niet op?!

Reply:
Er zat Partial op de hele type, die alles in the type optioneel maakt...